tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2359Test.php

```diff
@@ -14,7 +14,7 @@
 use Doctrine\ORM\Mapping\ClassMetadataFactory;
 use Doctrine\ORM\Mapping\Driver\MappingDriver;
 use Doctrine\Tests\DoctrineTestCase;
-use PHPUnit_Framework_MockObject_MockObject;
+use PHPUnit\Framework\MockObject\MockObject;
```

Comment:
It looks like this `use` of MockObject isn't used anywhere in the class.

Reply:
It's used in line 36 :)